.github/workflows/inductor.yml

```diff
@@ -120,11 +120,21 @@ jobs:
           { config: "cpu_inductor_timm", shard: 2, num_shards: 2, runner: "linux.12xlarge" },
           { config: "cpu_inductor_torchbench", shard: 1, num_shards: 2, runner: "linux.12xlarge" },
           { config: "cpu_inductor_torchbench", shard: 2, num_shards: 2, runner: "linux.12xlarge" },
+          { config: "cpu_inductor_huggingface_amp", shard: 1, num_shards: 1, runner: "linux.c7i.metal-24xl" },
+          { config: "cpu_inductor_timm_amp", shard: 1, num_shards: 2, runner: "linux.c7i.metal-24xl" },
+          { config: "cpu_inductor_timm_amp", shard: 2, num_shards: 2, runner: "linux.c7i.metal-24xl" },
+          { config: "cpu_inductor_torchbench_amp", shard: 1, num_shards: 2, runner: "linux.c7i.metal-24xl" },
+          { config: "cpu_inductor_torchbenc_amp", shard: 2, num_shards: 2, runner: "linux.c7i.metal-24xl" },
```

Comment:
```suggestion
          { config: "cpu_inductor_huggingface_amp", shard: 1, num_shards: 1, runner: "linux.16xlarge.spr" },
          { config: "cpu_inductor_timm_amp", shard: 1, num_shards: 2, runner: "linux.16xlarge.spr" },
          { config: "cpu_inductor_timm_amp", shard: 2, num_shards: 2, runner: "linux.16xlarge.spr" },
          { config: "cpu_inductor_torchbench_amp", shard: 1, num_shards: 2, runner: "linux.16xlarge.spr" },
          { config: "cpu_inductor_torchbenc_amp", shard: 2, num_shards: 2, runner: "linux.16xlarge.spr" },
```

Reply:
runner label refer to https://github.com/pytorch/test-infra/blob/main/.github/scale-config.yml#L36